lib/components/Menu.js

```diff
@@ -133,11 +133,18 @@ var Menu = module.exports = React.createClass({
     return options;
   },
 
+  buildName: function() {
+    var classes = 'Menu';
```

Comment:
Didn't think it was worth the extra dependency but now I see it really isn't extra... Pushed